numpy/core/src/umath/funcs.inc.src

```diff
@@ -267,6 +267,21 @@ npy_ObjectClip(PyObject *arr, PyObject *min, PyObject *max) {
     return o;
 }
 
+static PyObject *
+npy_ObjectPopCount(PyObject *obj) {
+    PyObject *result = NULL;
+
+    /* Try to use inbuilt popcount if available */
+    static PyObject *builtin_popcount_func = NULL;
+    builtin_popcount_func = PyObject_GetAttrString(obj, "bit_count");
```

Comment:
Hmm, most ufuncs already try to call a method of the same name when it exists?  There should be a shorthand to do this in `generate_umath` without the need to define this function.

Reply:
ohh yeah, let me see if we can auto-gen this, will make things easier.

Reply:
Ohh also makes sense now, why not all umath functions are here.

Reply:
Hey @seberg , I dug a bit deeper here and it seems all the `o` supported dtypes have a function in `funcs.inc.srs` and it does not seem automatic. Am I missing something here?

Reply:
Ah, there is the `"P"` character code that means: "Object, but just call the method".

Reply:
Ah nice, I missed it. Thanks! Added via [b86a322](https://github.com/numpy/numpy/pull/21429/commits/b86a322d45950469f3d84e5c05972212192e6ed0)